package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.0.0-development",
   "description": "The JSON Resume command line interface",
   "main": "index.js",
+  "type": "module",
```

Comment:
Should this be in the published package.json once we compile the source with Babel? Not sure anyone is importing resume-cli, but still.